tools/webinstall/install.sh

```diff
@@ -1188,6 +1207,104 @@ install_kraftkit() {
     fi
 }
 
+# install_dependencies_gnu installs all kraftkit dependencies needed for
+# building and running unikernels on gnu distributions.
+# Returns:
+# Code: 0 on success, 1 on error
+install_dependencies_gnu() {
+    _idd_list=$(printf "%s%s"                       \
+        "build-essential libncurses-dev flex git"   \
+        "wget socat bison unzip qemu-system"        \
```

Comment:
Can you please break this into multiple lines, alphabetically-sorted e.g.:
```
    _idd_list=$(printf "%s%s" \
        "bison" \ 
        "build-essential" \
        "flex" \
        "git" \
        "libncurses-dev" \
        "qemu-system" \
        "socat" \ 
        "unzip" \ 
        "wget" \ 
    )
```

Reply:
it will be a spam of '%s', but fine